ir/inst_other.go

```diff
@@ -164,6 +172,10 @@ type InstPhi struct {
 	Metadata
 }
 
+func (inst *InstPhi) Operands() []value.Value {
+	panic("implement me")
```

Comment:
@mewmew some of them, I have no idea what to do, left `panic`

Reply:
If there are no value operands, then return nil.

This is what I did for `phi` instructions, not sure if it makes sense or is the best option.

```go
case *ir.InstPhi:
	operands := make([]value.Value, 0, 2*len(inst.Incs))
	for _, inc := range inst.Incs {
		operands = append(operands, inc.X, inc.Pred)
	}
	return operands
```

See https://github.com/llir/llvm/issues/191#issuecomment-881198489 for comparison.